### ISSUE TITLE
URLs to video files seems broken

### ISSUE THREAD

**Issue by marlun** [leandromoreira/digital_video_introduction#63]
The video files that the setup script is trying to download does not seem to be available anymore?

**Comment by leandromoreira** [comment]
you're right, maybe it's a temporary thing otherwise i think we can host the file, it seems to be free to be used
